php/class-fieldmanager-field.php

```diff
@@ -939,7 +939,7 @@ public function get_remove_handle() {
 	 * @return string
 	 */
 	public function get_collapse_handle() {
-		return sprintf( '<div class="handlediv" title="%s"><br /></div>', esc_attr__( 'Click to toggle', 'fieldmanager' ) );
```

Comment:
Funny, so there was HTML markup for something, but it wasn't set to render anywhere?